media/js/mozorg/contact-spaces.js

```diff
@@ -160,6 +160,7 @@
                     // Hide spaces and community nav+meta
                     mozMap.toggleNav('contact');
                     mozMap.toggleMobileNav('contact');
+                    mozMap.setMapState();
```

Comment:
Hmm, I wonder if this is just masking over another bug perhaps - something that was added with the contact tab?